net/keepalived/Makefile

```diff
@@ -58,6 +58,7 @@ define Package/keepalived
     +libnl-genl \
     +libmagic \
     +libkmod \
+    +kmod-nf-ipvs \
```

Comment:
For what does we need this dependency?

Reply:
keepalived uses Linux kernel's ipvs module for the virtual server load balancing, and modules are already being loaded from the script, due to missing dependency, keepalive was crashing when virtual servers are configured.

https://github.com/openwrt/packages/blob/1aa0d0878039295ed5d65066b74de3d06d15115c/net/keepalived/files/keepalived.init#L509